Doc/whatsnew/3.8.rst

```diff
@@ -1138,6 +1138,10 @@ The new :func:`shlex.join` function acts as the inverse of :func:`shlex.split`.
 shutil
 ------
 
+Added :func:`shutil.link` and :func:`shutil.symlink` which allow for multiple
```

Comment:
Move this entry to to a blurb.  One is required. Since we are well before the 3.10 cutoff next May/June, maybe to 3.10.rst, but not required.

Reply:
@terryjreedy w.r.t the blank lines, I think that my additions to `shutil.py` are covered by:
> Surround top-level function and class definitions with two blank lines.

So, given this, I intend to revert the commit I made to reduce them to only one.

Please tell me if I've misunderstood or if you were referring to `test_shutil.py` instead, where I've used the following to group related tests:

> Extra blank lines may be used (sparingly) to separate groups of related functions.